dali/sasha/saxref.cpp

```diff
@@ -692,6 +692,7 @@ class CNewXRefManager: public CNewXRefManagerBase
     unsigned numuniqnodes = 0;
     Owned<IUserDescriptor> udesc;
     Linked<IPropertyTree> storagePlane;
+    bool isPlaneStriped;
```

Comment:
trivial: even though itialized in ctor, it's good practice for all members (for new code at least) to have member initializers. Helps if code is ever refactored etc., and compiler is usually smart enough not to initialize twice anyway.